ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/vsphere7/provision/cucushift-installer-rehearse-vsphere-ipi-vsphere7-provision-chain.metadata.json

```
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/ipi/vsphere7/provision/cucushift-installer-rehearse-vsphere-ipi-vsphere7-provision-chain.yaml",
```

```suggestion
	"path": "cucushift/installer/rehearse/vsphere/ipi/nested/provision/cucushift-installer-rehearse-vsphere-ipi-nested-provision-chain.yaml",
```
We should set the detailed nested vpshere version in job name rather than install workflow name.